app/src/main/res/values/strings.xml

```diff
@@ -1151,6 +1151,14 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_memorialType_stele_stone">Stone stele (memorial slab)</string>
     <string name="quest_memorialType_sculpture">Other sculpture</string>
 
+    <string name="quest_moped_prohibited_title">Are mopeds allowed on this cycleway?</string>
+    <string name="quest_moped_prohibited_designated">Yes, a mandatory sign is attached to the cycleway sign.</string>
+    <string name="quest_moped_prohibited_allowed">Yes,there is no prohibition sign is visible.</string>
+    <string name="quest_moped_prohibited_forbidden">No, a prohibition sign is attached to the cycleway sign.</string>
```

Comment:
"cycleway" is British English. The default strings are in American English. Use "bike path" instead.

As described earlier, I would find it better if it just asked for a sign because that doesn't presuppose any expert knowledge, i.e. in this case the legal situation in Belgium. Yes, I see that you worded the "allowed" answer in a way that lectures the user, but in my opinion, it is unnecessary to explain the legal situation to the user, it just makes the text longer. Speaking of which;

I find that the text options are too long. It's key to accommodate the keywords that also appear on the signs in the answer options, but in your current version, e.g. the designated and prohibited options have 10 words but differ in only one word. How about something like... 

**Is there a sign concerning mopeds for this bike path?**
- No, there is no sign
- Yes, mopeds are prohibited here
- Yes, this side path is mandatory for mopeds

(this is just a draft / suggestion)